frontend/homework/3/ques1.js

```diff
@@ -0,0 +1,22 @@
+const bills=[140,45,280];
+var tips=[];
+var total=[];
+
+function tipCalculator(arr){
```

Comment:
As you are calculating both the tips and the total amount so the function name must be generic